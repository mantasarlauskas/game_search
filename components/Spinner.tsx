import styles from 'styles/spinner.module.scss';

function Spinner() {
    return (
        <div className={styles.root} />
    );
}

export default Spinner;
