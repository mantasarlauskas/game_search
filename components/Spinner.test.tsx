import { render } from '@testing-library/react';
import Spinner from 'components/Spinner';

describe('<Spinner />', () => {
    it('matches snapshot', () => {
        const { container } = render(<Spinner />);
        expect(container).toMatchSnapshot();
    });
});
