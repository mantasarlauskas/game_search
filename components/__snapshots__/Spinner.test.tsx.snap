// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> matches snapshot 1`] = `
<div>
  <div
    class="root"
  />
</div>
`;
